src/Parser/Warlock/Destruction/Modules/Talents/Backdraft.js

```diff
@@ -83,7 +83,7 @@ class Backdraft extends Analyzer {
     );
   }
 
-  statisticOrder = STATISTIC_ORDER.OPTIONAL(0);
+  statisticOrder = STATISTIC_ORDER.CORE(5);
```

Comment:
The 'OPTIONAL' order is typically used for talents, which Backdraft apparently is. What's your justification for moving it to core?

Reply:
I have no idea how this works and this made it go where I wanted

Reply:
Ordering of stats boxes is left to right and then top to bottom, with some possible rearrangement based on box sizes. Core comes first, then Optional. It's generally been the style that Core is for core spells / mechanics, and Optional is for talents.

Changing these orders may have put these boxes where you wanted for the report you were testing with, but a player with different talents selected could mix things up again. (Due to some boxes being missing or others being added by the nature of which talents were selected)